README.md

```diff
@@ -20,6 +17,12 @@ To push image to Docker hub
 
 ` ./gradlew pushImage`
 
+
+## Clear production files 
+
+` ./gradlew clearProductionFiles`
```

Comment:
won't work, as far as `clearProductionFiles` is not a gradle task

Reply:
Resolved at https://github.com/emc-mongoose/console/pull/41/commits/618ac53ce3fbb7e3022a8d136dc19f46130428d6